src/chigame/games/admin.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import apps
+from django.contrib import admin
+
+app_models = apps.get_app_config("games").get_models()
+for model in app_models:
+    admin.site.register(model)
```

Comment:
I want you do register the models individually, since this way we will have more control over the customization features of each model.

https://docs.djangoproject.com/en/4.2/ref/contrib/admin/

https://realpython.com/customize-django-admin-python/



Then you can check out the above links to see how you can use the admin interface to customize your models